osmosis-1/osmosis-1.assetlist.json

```diff
@@ -12705,6 +12705,57 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/autism.png"
       }
+    },
+    {
```

Comment:
Apologies for the confusion; please do not edit this assetlist file. Only the osmosis.zone_assets.json file needs an entry.